db/models.py

```diff
@@ -50,3 +52,47 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Code Style: The `class Meta` is defined before all attributes of the `Order` model. According to the checklist, the `class Meta` should be defined after all attributes. Please move `class Meta` after the `user` attribute.